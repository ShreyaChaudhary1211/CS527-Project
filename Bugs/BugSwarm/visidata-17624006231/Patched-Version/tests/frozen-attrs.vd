sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample	Units		aggregate-col	sum	+	Add aggregator to current column
sample	Units		type-int		#	set type of current column to int
sample	Units		freeze-col		'	add a frozen copy of current column with all cells evaluated
sample			columns-sheet		Shift+C	open Columns Sheet: edit column properties for current sheet
