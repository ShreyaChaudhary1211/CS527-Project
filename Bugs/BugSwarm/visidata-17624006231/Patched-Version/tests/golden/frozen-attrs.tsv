name	width	height	type	fmtstr	formatter	displayer	value	expr	aggregators
OrderDate	12	1					2016-01-06	0	
Region	9	1					East	1	
Rep	10	1					Jones	2	
Item	9	1					Pencil	3	
Units	7	1	int				95	4	sum
Units_frozen	7	1	int				95	4	sum
Unit_Cost	11	1					1.99	5	
Total	9	1					189.05	6	
